src/Microsoft.AspNetCore.SystemWebAdapters/Internal/HttpValueCollection.cs

```diff
@@ -89,7 +89,7 @@ internal void FillFromString(string s, bool urlencoded = false, Encoding? encodi
 
     public override string ToString() => ToString(true);
 
-    private string ToString(bool urlencoded)
+    internal string ToString(bool urlencoded)
```

Comment:
This is because `_multiValue.ToString(false);` needs to be called in `HttpCookie`. This is also `internal` in the reference source: https://github.com/microsoft/referencesource/blob/51cf7850defa8a17d815b4700b67116e3fa283c2/System.Web/HttpValueCollection.cs#L294C38-L294C38